carthage-build.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+if [ "${1:?}" = "" ]; then
+    echo "Must set module name"
+    exit 1
+fi
+
+MODULE_NAME=${1:?}
```

Comment:
Can be written as `MODULE_NAME=${1:?"Must set module name"}`

Reply:
Fixed!